x/dex/keeper/core.go

```diff
@@ -605,6 +607,7 @@ func (k Keeper) WithdrawFilledLimitOrderCore(
 		coinTakerDenomOut := sdk.NewCoin(tradePairID.TakerDenom, amountOutTokenOut)
 		coinMakerDenomRefund := sdk.NewCoin(tradePairID.MakerDenom, remainingTokenIn)
 		coins := sdk.NewCoins(coinTakerDenomOut, coinMakerDenomRefund)
+		ctx.EventManager().EmitEvents(types.GetEventsWithdrawnAmount(coins))
```

Comment:
For consistency, I would just track CoinTakerDenomOut. coinMakerDenomRefund is the same coin they would get if they cancel the LO, which we don't track.